internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,60 @@
 package daysteps
 
 import (
-	...
-)
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
 
-const (
-	StepLength = 0.65
+	"github.com/Evrard-ro/Go-fit5/internal/personaldata"
+	"github.com/Evrard-ro/Go-fit5/internal/spentenergy"
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	pars := strings.Split(datastring, ",")
+	if len(pars) != 2 {
+		return fmt.Errorf("invalid data format: expected 2 parts, got %d", len(pars))
+	}
+	ds.Steps, err = strconv.Atoi(pars[0])
+	if err != nil {
+		return fmt.Errorf("failed to parse steps: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("failed to parse steps: %w", err)
```